packages/patrol/example/ios/fastlane/Fastfile

```diff
@@ -0,0 +1,30 @@
+default_platform(:ios)
+
+git_authorization = "#{ENV["GIT_USERNAME"]}:#{ENV["GIT_TOKEN"]}"
```

Comment:
If you require `GIT_USERNAME` and `GIT_TOKEN`, then I recommend using `ENV.fetch("GIT_USERNAME")` and `ENV.fetch("GIT_TOKEN")`, so it'll fail immediately when these credentials are not found.